docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,235 @@
+
+```shell
+$ kubectl get pods --namespace=kube-system
+NAME                                           READY     STATUS    RESTARTS   AGE
+....
+my-scheduler-lnf4s-4744f                       1/1       Running   0          2m
+...
+```
+
+You should see a "Running" my-scheduler pod, in addition to the default kube-scheduler
+pod in this list.
+
+### 4. Specify schedulers for pods
+
+Now that our second scheduler is running, let's create some pods, and direct them to be scheduled by either the default scheduler or the one we just deployed. In order to schedule a given pod using a specific scheduler, we specify the name of the
+scheduler as an annotation in that pod spec. Let's look at three examples.
+
+
+1. Pod spec without any scheduler annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: no-annotation
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-no-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  When no scheduler annotation is supplied, the pod is automatically scheduled using the
+  default-scheduler.
+
+  Save this file as `pod1.yaml` and submit it to the Kubernetes cluster.
+
+  ```shell
+  $ kubectl create -f pod1.yaml
+  ```
+2. Pod spec with `default-scheduler` annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: annotation-default-scheduler
+    annotations:
+      scheduler.alpha.kubernetes.io/name: default-scheduler
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-default-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  A scheduler is specified by supplying the scheduler name as a value to the annotation
+  with key `scheduler.alpha.kubernetes.io/name`. In this case, we supply the name of the
+  default scheduler which is `default-scheduler`.
+
+  Save this file as `pod2.yaml` and submit it to the Kubernetes cluster.
+
+  ```shell
+  $ kubectl create -f pod2.yaml
+  ```
+3. Pod spec with `my-scheduler` annotation
+
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: annotation-second-scheduler
+    annotations:
+      scheduler.alpha.kubernetes.io/name: my-scheduler
+    labels:
+      name: multischeduler-example
+  spec:
+    containers:
+    - name: pod-with-second-annotation-container
+      image: gcr.io/google_containers/pause:2.0
+  ```
+
+  In this case, we specify that this pod should be scheduled using the scheduler that we
+  deployed - `my-scheduler`. Note that the value of the annotation with key
+  `scheduler.alpha.kubernetes.io/name` should match the name supplied to the scheduler
+  command as an argument in the deployment config for the scheduler.
+
+  Save this file as `pod3.yaml` and submit it to the Kubernetes cluster.
+
+  ```shell
+  $ kubectl create -f pod3.yaml
+  ```
+
+  Verify that all three pods are running.
+
+  ```shell
+  $ kubectl get pods
```

Comment:
For examples where it's just a simple command, leave the "$" out. For examples where it's a mix of the command and output from the command, leave the "$" in.